### PR TITLE
Nfdiv 721

### PR DESCRIPTION
**Pull request by velimd**
### JIRA link ###
https://tools.hmcts.net/jira/browse/NFDIV-721
Screenshot 2021-07-06 at 15-53-57 Apply for a divorce - Check your husband's answers - GOV UK

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/app/case/answers/getAnswerRows.ts [hunk 1]
```diff
@@ -23,7 +27,12 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
     formState: Partial<Case>;
   } = this.ctx;
 
-  const stepsWithContent = isApplicant2 ? stepsWithContentApplicant2 : stepsWithContentApplicant1;
+  let stepsWithContent = isApplicant2 ? stepsWithContentApplicant2 : stepsWithContentApplicant1;
+  if (overrideStepsContent === 1) {
+    stepsWithContent = stepsWithContentApplicant1;
```
Comment (tomxelliott) [inline src/main/app/case/answers/getAnswerRows.ts:32]: Do you need to assign `stepsWithContentApplicant1` to `stepsWithContent` again here? What does overrideStepsContent do that the isApplicant2 check doesn't?

File: src/main/app/case/answers/getAnswerRows.ts [hunk 2]
```diff
@@ -23,7 +27,12 @@ export const getAnswerRows = function (section: Sections): GovUkNunjucksSummary[
...
+  } else if (overrideStepsContent === 2) {
+    stepsWithContent = stepsWithContentApplicant2;
```
Comment (tomxelliott) [inline src/main/app/case/answers/getAnswerRows.ts:34]: As above

**Review by tomxelliott — APPROVED** [review]
Looks good to me, just a small comment

File: src/main/app/case/answers/getAnswerRows.ts (hunk 1 above)
Comment (velimd) [reply src/main/app/case/answers/getAnswerRows.ts:32]: As they are app2, theyll only get app2 content displaying but check joint page wants app1 answers, hence i thought of this override variable.

File: src/main/app/case/answers/getAnswerRows.ts (hunk 2 above)
Comment (velimd) [reply src/main/app/case/answers/getAnswerRows.ts:34]: i guess i thought ill check for both incase app1 ever needed to view app2 answers.

**Review by lara-mpt — APPROVED** [review]

**Pull request merged**